tests/lmeval/test_lmeval.py

```diff
@@ -127,7 +127,12 @@ def test_lm_eval(self, test_data_file: str):
         base_results = self._eval_base_model()
 
         if not self.save_dir:
-            self.save_dir = self.model.split("/")[1] + f"-{self.scheme}"
+            base_name = self.model.split("/")[1] + f"-{self.scheme}"
+            hf_cache = os.environ.get("HF_HUB_CACHE")
+            if hf_cache and os.path.isdir(hf_cache):
+                self.save_dir = os.path.join(hf_cache, base_name)
+            else:
+                self.save_dir = base_name
```

Comment:
Is this compatible with the format in which models are downloaded from HF hub? They have their own naming structure, e.g. a folder `models--deepseek-ai--DeepSeek-V3` for model id `deepseek-ai/DeepSeek-V3`. but maybe there's a helper in HF's cli codebase to help with this?